api/end_device.proto

```diff
@@ -161,15 +161,13 @@ message EndDeviceVersion {
   uint64 min_frequency = 9;
   // Maximum frequency the device is capable of using (Hz).
   uint64 max_frequency = 10;
-  // Whether the device resets the frame counters (not LoRaWAN compliant).
-  bool resets_f_cnt = 11;
```

Comment:
I'm really not a big fan of this kind of out-of-scope changes, @rvolosatovs. This PR was about validation...

Reply:
We're working on the API layer here and this directly influences this PR